### PR TITLE
create CONTRIBUTING-te.md

### PR DESCRIPTION
**Pull request by repsick3r**
## What does this PR do?
Added `CONTRIBUTING-te.md`

### Description
Added `CONTRIBUTING-te.md`

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
add the link to this doc from the cocs/README. we want to add the (link+doc) for each doc at the same time. Thanks!

**Comment by repsick3r** [comment]
> add the link to this doc from the cocs/README. we want to add the (link+doc) for each doc at the same time. Thanks!

got it.

File: docs/CONTRIBUTING-te.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    ```text
+    BAD : * [Another Awesome Book](http://example.com/)- John Doe (HTML)
+    BAD : * [Another Awesome Book](https://downloads.example.org/book.html)- John Doe (download site)
+    GOOD: * [Another Awesome Book](http://example.com/) - John Doe (HTML) [(PDF, EPUB)](https://downloads.example.org/book.html)
+    ```
+
+- పాత పుస్తకాల శీర్షికలో ప్రచురణ సంవత్సరాన్ని చేర్చండి:
+
+    ```text
+    BAD : * [A Very Awesome Book](https://example.org/book.html) - Jane Roe - 1970
```
Comment (shrn01) [inline docs/CONTRIBUTING-te.md:127]: BAD/GOOD ni telugu lo rayagalamaa

**Comment by eshellman** [comment]
this can be closed I think?

**Comment by shrn01** [comment]
The translation looks good only. We just have to remove the Code-of-conduct.md link from the readme file

File: docs/CONTRIBUTING-te.md (hunk 1 above)
Comment (repsick3r) [reply docs/CONTRIBUTING-te.md:127]: We can do that, but the thing is i have took some other translations as reference, where translation was omitted completely in the code parts between ``` and ```.

So what should i do.

reference : https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING-sv.md

File: docs/README.md [hunk 2]
```diff
@@ -99,6 +99,8 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
   - [How-to](HOWTO-sv.md)
 - Tamil / தமிழ்
 - Telugu / తెలుగు
+  - [Code of Conduct](CODE_OF_CONDUCT-te.md)
```
Comment (shrn01) [inline docs/README.md:102]: Please remove this entry

Comment (repsick3r) [reply docs/README.md]: Done.

File: docs/README.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
   - [How-to](HOWTO-sv.md)
 - Tamil / தமிழ்
 - Telugu / తెలుగు
-  - [Code of Conduct](CODE_OF_CONDUCT-te.md)
```
Comment (shrn01) [inline docs/README.md:103]: Looks like this entry was added from another PR, please add it back

**Comment by eshellman** [comment]
> The translation looks good only. We just have to remove the Code-of-conduct.md link from the readme file

OK, I guess it was another translation that we just added.

File: docs/README.md (hunk 3 above)
Comment (repsick3r) [reply docs/README.md:103]: ok

**Pull request merged**